src/style.css

```
@@ -1 +1,72 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
```

```suggestion
  font-family: "Roboto", sans-serif;
  font-weight: 500;
```
always set the font in `html {...}` tag and remove `font-style: normal;` as it is default value